runtime/Python2/src/antlr4/TokenStreamRewriter.py

```diff
@@ -100,15 +100,17 @@ def getProgram(self, program_name):
         return self.programs.setdefault(program_name, [])
 
     def getDefaultText(self):
-        return self.getText(self.DEFAULT_PROGRAM_NAME, Interval(0, len(self.tokens.tokens)))
+        return self.getText(self.DEFAULT_PROGRAM_NAME, Interval(0, len(self.tokens.tokens) - 1))
```

Comment:
although range check is performed in ``TokenStreamRewriter.getText``, this line is misleading.